conferences/abstracts/OSF19_Poster_submission.md

```diff
@@ -0,0 +1,82 @@
+# OSFair2019 Poster Submission  
+Open Science Fair 2019, September 16-18, Porto, Portugal
+
+*For Authors: remove text in italics and replace with your own content.*
+
+## The Turing Way: A handbook for reproducible data science 
+
+*Author 1 name, affiliation, email address; Author 2 name, affiliation, email address*
+
+The Turing Way, Alan Turing Institute; Rachael Ainsworth, University of Manchester, rachael.ainsworth@manchester.ac.uk
```

Comment:
For posters, please can we have all the authors. Fine for now to list the authors who were on the poster I made (https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/presentations/TPS_ScopingWorkshop_20190321/Whitaker_TuringWayPoster_TPSScopingWorkshop.pdf) although if you see key people who should also be authors please add them.